src/skyrimvrsavegame.cpp

```diff
@@ -2,72 +2,120 @@
 
 #include <Windows.h>
 
-SkyrimVRSaveGame::SkyrimVRSaveGame(QString const &fileName, MOBase::IPluginGame const *game, bool const lightEnabled) :
-  GamebryoSaveGame(fileName, game, lightEnabled)
+#include "gameskyrimvr.h"
+
+SkyrimVRSaveGame::SkyrimVRSaveGame(QString const &fileName, GameSkyrimVR const *game) :
+  GamebryoSaveGame(fileName, game, true)
 {
-    FileWrapper file(this, "TESV_SAVEGAME"); //10bytes
-    unsigned long headerSize;
-    file.read(headerSize); // header size "TESV_SAVEGAME"
-    file.skip<unsigned long>(); // header version 74. Original Skyrim is 79
-    file.read(m_SaveNumber);
+  FileWrapper file(fileName, "TESV_SAVEGAME"); //10bytes
+
+  unsigned long version;
+  FILETIME ftime;
+  fetchInformationFields(file, version, m_PCName, m_PCLevel, m_PCLocation, m_SaveNumber, ftime);
 
-    file.read(m_PCName);
+  //A file time is a 64-bit value that represents the number of 100-nanosecond
+             //intervals that have elapsed since 12:00 A.M. January 1, 1601 Coordinated Universal Time (UTC).
+             //So we need to convert that to something useful
 
-    unsigned long temp;
-    file.read(temp);
-    m_PCLevel = static_cast<unsigned short>(temp);
+             //For some reason, the file time is off by about 6 hours.
+             //So we need to subtract those 6 hours from the filetime.
+  _ULARGE_INTEGER time;
+  time.LowPart = ftime.dwLowDateTime;
+  time.HighPart = ftime.dwHighDateTime;
+  time.QuadPart -= 2.16e11;
+  ftime.dwHighDateTime = time.HighPart;
+  ftime.dwLowDateTime = time.LowPart;
 
-    file.read(m_PCLocation);
+  SYSTEMTIME ctime;
+  ::FileTimeToSystemTime(&ftime, &ctime);
+
+  setCreationTime(ctime);
+}
 
-    QString timeOfDay;
-    file.read(timeOfDay);
 
-    QString race;
-    file.read(race); // race name (i.e. BretonRace)
+void SkyrimVRSaveGame::fetchInformationFields(
+  FileWrapper& file,
+  unsigned long& version,
+  QString& playerName,
+  unsigned short& playerLevel,
+  QString& playerLocation,
+  unsigned long& saveNumber,
+  FILETIME& creationTime) const
+{
+  unsigned long headerSize;
+  file.read(headerSize); // header size "TESV_SAVEGAME"
+  file.read(version);
```

Comment:
The comment got lost here:
`// header version 74. Original Skyrim is 79` 
It might be useful info to know